console.rst

```diff
@@ -45,9 +45,16 @@ want a command to create a user::
 
         protected function execute(InputInterface $input, OutputInterface $output)
         {
-            // ...
+            // ... put here the code to run in your command
+
+            // this method must return an integer number with the "exit status code"
```

Comment:
Maybe you could explain a little more that any non-0 exit code is often seen as a "fail". And so the 0 and 1 shown here are only examples.

https://www.gnu.org/software/bash/manual/html_node/Exit-Status.html#:~:text=A%20non%2Dzero%20exit%20status,N%20as%20the%20exit%20status.

Reply:
woops didn't mean to anger carsonbot 😇

Reply:
You are right, i read your link for find a better explication for exit code.